api4/post.go

```diff
@@ -576,6 +577,28 @@ func patchPost(c *Context, w http.ResponseWriter, r *http.Request) {
 	w.Write([]byte(patchedPost.ToJson()))
 }
 
+func setPostUnread(c *Context, w http.ResponseWriter, r *http.Request) {
+	c.RequirePostId()
+	if c.Err != nil {
+		return
+	}
+	if c.App.Session.UserId != c.Params.UserId && c.App.SessionHasPermissionToUser(c.App.Session, c.Params.UserId) {
```

Comment:
Wait this should be `!c.App.SessionHasPermissionToUser(c.App.Session, c.Params.UserId)` right?

Reply:
good catch!